app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt

```diff
@@ -199,17 +208,31 @@ private fun OngoingCallContent(
 @Composable
 private fun OngoingCallTopBar(
     conversationName: String,
+    isCbrEnabled: Boolean,
     onCollapse: () -> Unit
 ) {
-    WireCenterAlignedTopAppBar(
-        onNavigationPressed = onCollapse,
-        titleStyle = MaterialTheme.wireTypography.title02,
-        maxLines = 1,
-        title = conversationName,
-        navigationIconType = NavigationIconType.Collapse,
-        elevation = 0.dp,
-        actions = {}
-    )
+    Column {
+        WireCenterAlignedTopAppBar(
+            onNavigationPressed = onCollapse,
+            titleStyle = MaterialTheme.wireTypography.title02,
+            maxLines = 1,
+            title = conversationName,
+            navigationIconType = NavigationIconType.Collapse,
+            elevation = 0.dp,
+            actions = {}
+        )
+        if (isCbrEnabled) {
+            Text(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .offset(y = -(5).dp),
```

Comment:
Can we also extract this to a val? Otherwise detekt will prob complain about the Magic Numbers